EpsilonCCS/Src/main.c

```diff
@@ -357,29 +354,26 @@ static void MX_CAN1_UserInit(void)
     static CanRxMsgTypeDef rxMessage;
     hcan1.pTxMsg = &txMessage;
     hcan1.pRxMsg = &rxMessage;
-    // hcan1.pTxMsg->StdId = 0x0; // CAN message address, set in Lights.c
+    // hcan1.pTxMsg->StdId = 0x0; // CAN message address
     hcan1.pTxMsg->ExtId = 0x0; // Only used if (hcan1.pTxMsg->IDE == CAN_ID_EXT)
     hcan1.pTxMsg->RTR = CAN_RTR_DATA; // Data request, not remote request
     hcan1.pTxMsg->IDE = CAN_ID_STD; // Standard CAN, not Extended
-    hcan1.pTxMsg->DLC = 1; // Data size in bytes
+    // hcan2.pTxMsg->DLC = ; // Value is variable
```

Comment:
Within the MBED code it was not variable, everything was 8 bytes always. I am pretty sure all Tritium stuff is a constant 8 bytes as well.  Within the "UofC Solar Car Team Communications Protocol" I don't see anything indicating non-standard CAN byte sizes, if they do exist then the document should be updated for that.

Same comment for CAN2 DLC